administration/src/components/cards/CreateCardsController.tsx

```diff
@@ -35,14 +37,23 @@ const CreateCardsController = () => {
   const confirm = async () => {
     try {
       setState(CardActivationState.loading)
+
       const activationCodes = cardBlueprints.map(cardBlueprint => {
         return cardBlueprint.generateActivationCode()
-      })
+      })      
+      const staticCodes = projectConfig.staticQrCodesEnabled ? cardBlueprints.map(cardBlueprints => {
+        return cardBlueprints.generateStaticVerifyCode()
+      }) : null
+            
+      const font = await loadTTFFont('NotoSans', 'normal', '/pdf-fonts/NotoSans-Regular.ttf')
+      const pdfDataUri = generatePdf(font, region, activationCodes, staticCodes)
```

Comment:
This rearrangement should fix that cards are created, even though PDF generation fails.